indicators/NO_FUNC_001-004/R/NO_FUNC_001-004.qmd

```diff
@@ -0,0 +1,3126 @@
+  - name: Erik Framstad
+    affiliations:
+      - ref: myID               # To reuse affiliations referecen the id like this
+date: April 9, 2025 # Enter date 
+callout-icon: false
+lightbox: true
+css: ../../../style.css
+code-links:
+      - text: Add a review
+        icon: github
+        href: https://github.com/NINAnor/ecRxiv
+---
+
+<!--# This is a template for how to document the indicator analyses. Make sure also to not change the order, or modify, the headers, unless you really need to. This is because it easier to read if all the indicators are presented using the same layout. If there is one header where you don't have anything to write, just leave the header as is, and don't write anything below it. If you are providing code, be careful to annotate and comment on every step in the analysis. Before starting it is recommended to fill in as much as you can in the metadata file. This file will populate the initial table in your output.-->
+
+<!--# Load all you dependencies here -->
+
+```{r setup}
+#| include: false
+library(knitr)
+library(tidyverse)
+library(kableExtra)
+library(here)
+library(sf)
+library(readxl)
+library(plyr)
+library(dplyr)
+library(RColorBrewer)
+library("gridExtra") 
+library(ggridges)
+library(tmap)
+library(tmaptools)
+library(raster)
+library(stars)
+library(betareg)
+library(glmmTMB)
+library(flextable)
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r source}
+#| echo: false
+source(here::here("_common.R"))
+```
+
+```{r}
+#| echo: false
+meta <- readxl::read_xlsx("../metadata.xlsx")
+st <- meta |>
+  filter(Variable == "status") |>
+  pull(Value)
+version <- meta |>
+  filter(Variable == "Version") |>
+  pull(Value)
+auth <- meta |>
+  filter(Variable == "authors") |>
+  pull(Value)
+year <- meta |>
+  filter(Variable == "yearAdded") |>
+  pull(Value)
+id <- meta |>
+  filter(Variable == "indicatorID") |>
+  pull(Value)
+name <- meta |>
+  filter(Variable == "indicatorName") |>
+  pull(Value)
+url <- meta |>
+  filter(Variable == "url") |>
+  pull(Value)
+
+meta <- meta |>
+  mutate(Variable = case_match(Variable,
+    "indicatorID" ~ "Indicator ID" ,
+    "indicatorName" ~ "Indicator Name",
+    "country" ~ "Country",
+    "continent" ~ "Continent",
+    "ECT" ~ "Ecosystem Condition Typology Class",
+    "yearAdded" ~ "Year added",
+    "yearLastUpdate" ~ "Last update",
+    .default = Variable
+   )
+  ) |>
+  filter(Variable != "authors")
+
+```
+
+<!--# The following parts are autogenerated. Do not edit. -->
+
+```{r}
+#| echo: false
+#| results: asis
+status(st)
+```
+
+::: {layout-ncol="2"}
+
+
+
+> **Recomended citation**: `r paste(auth, " ", year, ". ", name, " (ID: ", id, ") ", "v. ", version, ". ecRxiv: ", url, sep="")`
+
+> **Version**: `r version`
+
+:::
+
+```{=html}
+<details>
+<summary>Show metadata</summary>
+```
+
+```{r tbl-meta}
+#| tbl-cap: 'Indicator metadata'
+#| echo: false
+#| warning: false
+
+meta |>
+  select(Variable, Value) |>
```

Comment:
I had an issue with the select here. It seems to be conflicting packages. Not sure if others have the same problem. If yes, then a solution can be to use the conflicted package, which helps to choose which package should be used. Otherwise, you can also use dplyr::select. If it's not an issue for others, ignore this comment.

Reply:
I can also check, but in any case it should work on all systems, so good suggestion to add this to the setup block:
`conflicted::conflict_prefer("select", "dplyr")`